### ISSUE TITLE
[BUG] Energy Unextractable

### ISSUE THREAD

**Issue by Benji07p** [Frinn38/Custom-Machinery#89]
Hello, i've create a custom machinery that generate energy from the hydrogen (chemlib).
The machinery generate the energy correctly, but i can't extract it.
The energy stay in the machinery.
I'm using custom machinery version 1.18.2-0.7.6 and kubejs-forge-1802.5.5-build.569.
Maybe it's me who's just misunderstood.
[antimatter_generator.json.txt](https://github.com/Frinn38/Custom-Machinery/files/11301571/antimatter_generator.json.txt)
[antimatter.js.txt](https://github.com/Frinn38/Custom-Machinery/files/11301572/antimatter.js.txt)

**Comment by Frinn38** [comment]
By default the energy component only allow to input energy, initially the default was to allow both input and output but that caused issues (machine would send energy to nearby cables then take it back).
If you want to to set the energy component to output you need to specify a side config : [Wiki](https://frinn.gitbook.io/custom-machinery-1.18/misc/config)
TL;DR : add the following property to your energy component json : 
```json
"config": {
    "default": "output"
}
```
With that the machine will output energy on all sides, if you also need to input energy you will have to configure it per side (c.f the wiki).
If you still need help with your machine consider joining the discord (link in github and Curseforge descriptions) as it will be easier to help you there than in Github issues.
